libs/sdk-ui-dashboard/src/_staging/layout/sizing.ts

```diff
@@ -352,3 +356,142 @@ export const getDashboardLayoutItemHeight = (size: IDashboardLayoutSize): number
 
 export const getDashboardLayoutItemHeightForGrid = (gridHeight: number): number =>
     gridHeight * GRID_ROW_HEIGHT_IN_PX;
+
+export const determineSizeForScreen = (
+    screen: ScreenSize,
+    layoutItemSize?: IDashboardLayoutSizeByScreenSize,
+): IDashboardLayoutSize => {
+    return {
+        ...(layoutItemSize ? layoutItemSize[screen] : {}),
+        gridWidth: determineWidthForScreen(screen, layoutItemSize),
+    };
+};
+
+export const determineWidthForScreen = (
+    screen: ScreenSize,
+    layoutItemSize?: IDashboardLayoutSizeByScreenSize,
+) => {
+    // Determine if element has size set in metadata object for the current screen size
+    const providedSizeForScreen = layoutItemSize ? layoutItemSize[screen] : undefined;
```

Comment:
`const providedSizeForScreen = layoutItemSize.?[screen];` is shorter (this is probably older code)

Reply:
fixed